src/vshaxe/HaxeCommand.hx

```diff
@@ -31,6 +31,7 @@ enum abstract HaxeCommand(String) to String {
 	var Cache_CopyNodeValue = command("cache.copyNodeValue");
 	var Cache_ReloadNode = command("cache.reloadNode");
 	var Cache_GotoNode = command("cache.gotoNode");
+	var Type = "type";
```

Comment:
I wouldn't necessarily add this here, since it's not a custom Haxe command, but a built-in one. All the other ones have the `"haxe."` prefix.